### PR TITLE
Update Chef dependency to >= 16

### PR DESCRIPTION
**Pull request by TheREK3R**

### CODE REVIEW

File: Rakefile [hunk 1]
```diff
@@ -122,14 +122,6 @@ task :unit do
...
 end
 
-desc 'Multi-node Integration with Vagrant'
-task :multi_node do
-  run_command('rm -rf cookbooks nodes Berksfile.lock')
-  run_command('berks vendor cookbooks')
-  run_command('mkdir nodes')
-  run_command('vagrant up')
-end
-
```
Comment (ramereth) [inline Rakefile:132]: Why was this removed?

Comment (TheREK3R) [reply Rakefile:132]: IDK, I think in response to cookstyle, but it was probably an oversite by me, ill fix it

Comment (TheREK3R) [reply Rakefile:132]: @ramereth this is a github glitch. This change is already in master

Comment (ramereth) [reply Rakefile:132]: Nope, it shows up in 7435397b4e81082a1d03cb62767807e14c39fc2b

Comment (TheREK3R) [reply Rakefile:132]: Oh my goodness 🤦 I was looking at osl-ganeti so it didnt show up in my git diff

**Comment by ramereth** [comment]
!bump major ~

**Comment by osuosl-manatee** [comment]
Jenkins has merged this PR into `master` and has automatically performed a major-level version bump to v4.0.0.  Have a nice day!
